<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1445</width>
    <height>704</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btnRigolDiscover">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>R. Discover</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStart">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRigolConnect">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>R. Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRigolDisconnect">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>R. Disconnect</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblStatus">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>531</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblReply">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>531</width>
      <height>71</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnIDN">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>IDN ?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLastError">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>280</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Last Error</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblError">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>580</y>
      <width>531</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClearReply">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>440</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Reply</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ledTestCommand">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>370</y>
      <width>521</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSendTextCmd">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>360</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Send It</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnESR_Q">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>ESR ?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnESE_Q">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>70</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>ESE ?</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ledNumericValue">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>70</y>
      <width>241</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblNumericValue">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>110</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>Numeric Value</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnESE_S">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>120</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Set ESE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSRE_Q">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>70</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>SRE ?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSRE_S">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>120</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Set SRE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSTB_Q">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>70</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>STB ?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnATT_Q">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>70</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>ATT ?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnATT_S">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>120</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Set ATT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLEV_Q">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>70</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>LEV ?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLEV_S">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>120</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Set LEV</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMODE_S">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>120</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Set MODE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMODE_Q">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>70</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>MODE ?</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblStringValue">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>190</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>String Value</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ledStringValue">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>150</y>
      <width>241</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFREQ_Q">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>FREQ ?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFREQ_S">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Set FREQ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnON_OFF_S">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>220</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Set RF ON/OFF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnON_OFF_Q">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>RF ON/OFF ?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAM_DEPT_Q">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>170</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>AM DEPT ?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAM_DEPT_S">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>220</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Set AM DEPT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAM_FREQ_Q">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>170</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>AM FREQ ?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAM_FREQ_S">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>220</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Set AM FREQ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAM_STAT_Q">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>170</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>AM STAT ?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAM_STAT_S">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>220</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Set AM STAT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMOD_STAT_Q">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>170</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>MOD STAT ?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMOD_STAT_S">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>220</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Set MOD STAT</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1445</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
